Sources/MapboxCoreNavigation/EHorizon/EHorizonObjectEnterExitInfo.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import MapboxNavigationNative
+
+public struct EHorizonObjectEnterExitInfo {
+
+    /** Road object id */
+    public let roadObjectId: String
+
+    /**
+     If object was entered via it's start for `onRoadObjectEnter`
+     or if object was exited via it's end for `onRoadObjectExit`
+     */
+    public let isEnterFromStartOrExitFromEnd: Bool
```

Comment:
Unlike the other two properties, this property doesn’t appear to be intrinsic to the road object itself, just the context in which the object happens to be used. So I think it would be appropriate to omit this property from the struct and include it alongside the struct as a separate argument to the delegate method.

Reply:
I ended up removing this struct as part of transitioning to a notification system – now this information sits alongside the other information in a user info dictionary.